lib/LanguagesModes/common_modes.dart

```diff
@@ -1,71 +1,139 @@
 import 'package:highlight/highlight_core.dart';
```

Comment:
Added redundant types.
Formatting is a bit better but still suffers without trailing commas.
Will require more work, no benefit of merging this.